Package.resolved

```diff
@@ -6,8 +6,8 @@
         "repositoryURL": "https://github.com/mattpolzin/OpenAPIKit.git",
         "state": {
           "branch": null,
-          "revision": "d40e6c3d57b1b27ac2114861d5a83cf8ee9d206d",
-          "version": "3.0.0-alpha.6"
+          "revision": "e805233e54afd394937cecb0a28e575b3388f60f",
+          "version": "3.0.0-alpha.9"
```

Comment:
In the Package.swift, we should require `3.0.0-alpha.9` and above as the code is now incompatible with earlier alphas.

Reply:
Agreed